### PR TITLE
Implementation of a menu-driven Priority Queue using Array in C++

### LINKED ISSUE #2422: Priority Queue in C
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)
Yes
## Motivation

(Please outline the motivation for the proposal.)
To Implement a Priority Queue using arrays in C
## Pitch

(Please explain why this feature should be implemented and how it would be used.)
This is a very potential question in coding. Being a part of the GSSOC'21, I would like to implement this in C .
Please do assign this to me

### PR DESCRIPTION
**Pull request by Suvraneel**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yep !

### Description
Implementation of a dynamic, menu-driven Priority Queue using Array in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

resolves #2422 
Assigned by @HarshCasper

### CODE REVIEW

**Comment by Suvraneel** [comment]
Hello maintainers!
Kindly review this.... I've followed all norms/changes requested in my previous PRs regarding inline comments, new line chars etc. :)
@HarshCasper @iamrajiv
@bhav09 @siddharth25pandey

-Thanks

File: C-Plus-Plus/ds/priorityQ_array.cpp [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+3       ---->   (50)
+4       ---->   (75)
+
+Enter choice:   3
+Element with maximum priority in queue: 
+(75) with priority: 4
+
+Enter choice:   6
+Program terminated successfully.
+*/
```
Comment (bhav09) [inline C-Plus-Plus/ds/priorityQ_array.cpp:219]: Can you please add a new line here.

**Review by bhav09 — COMMENTED** [review]
Reviewed.

**Comment by bhav09** [comment]
Well done, @Suvraneel :)

**Comment by Suvraneel** [comment]
Thanks ! @bhav09 :))

**Pull request merged**